billing.go

```diff
@@ -66,10 +64,8 @@ func (c *Client) UpdateBillingPlan(planId string, pathValues map[string]map[stri
 			Value:     data,
 		})
 	}
-
-	jsonData, err := json.Marshal(patchData)
-	buf := bytes.NewBuffer(jsonData)
-	req, err := c.NewRequest(http.MethodPatch, fmt.Sprintf("%s%s%s", c.APIBase, "/v1/payments/billing-plans/", planId), buf)
+	
+	req, err := c.NewRequest(http.MethodPatch, fmt.Sprintf("%s%s%s", c.APIBase, "/v1/payments/billing-plans/", planId), patchData)
```

Comment:
I believe that the previous format was correct.

Reply:
Facts speak louder than words

Reply:
Now I can see it! Thanks a lot for the fix!